make/Images.gmk

```diff
@@ -110,13 +114,22 @@ $(eval $(call SetupExecute, jlink_jdk, \
 
 JLINK_JDK_TARGETS := $(jlink_jdk)
 
+ifeq ($(BUILD_OPENJCEPLUS), false)
+  LEGACY_JRE_POST_COMMAND :=
+else ifeq ($(call isTargetOs, windows), true)
+  LEGACY_JRE_POST_COMMAND := $(CP) -rf $(SUPPORT_OUTPUTDIR)/modules_cmds/openjceplus/C $(JRE_IMAGE_DIR)/bin
+else
+  LEGACY_JRE_POST_COMMAND := $(CP) -rf $(SUPPORT_OUTPUTDIR)/modules_libs/openjceplus/C $(JRE_IMAGE_DIR)/lib
+endif
```

Comment:
After discovering that `StripNativeDebugSymbolsPlugin` only applies on Linux, this can be simplified:
```
ifeq ($(call And, $(BUILD_OPENJCEPLUS) $(call isTargetOs, linux)), true)
  LEGACY_JRE_POST_COMMAND := $(CP) -rf $(call FindLibDirForModule, openjceplus)/C $(JRE_IMAGE_DIR)/lib
else
  LEGACY_JRE_POST_COMMAND :=
endif
```
but perhaps tweaking `StripNativeDebugSymbolsPlugin` to avoid a need for this is a better path forward.

Reply:
OK will pursue a fix by changing the code in `StripNativeDebugSymbolsPlugin` instead of the makefiles themselves. Advantage certainly being that all users of `jlink` will avoid the stripping of the library.

Reply:
Updated the code according to this conversation.